codespell_lib/data/dictionary.txt

```diff
@@ -2546,28 +2840,51 @@ counterfit->counterfeit
 countires->countries
 countour->contour
 countours->contours
+countr->counter
 countrie's->countries, countries', country's,
+cource->course
 coururier->courier, couturier,
 couse->course, cause,
+couses->courses, causes,
+cousing->cousin
+couted->counted
 couterpart->counterpart
 coutner->counter
 coutners->counters
+covere->cover
+coveres->covers
 coverted->converted, covered, coveted,
+coverting->converting
+covnert->convert
+coyp->copy
 coypright->copyright
+cpation->caption
+cpoy->copy
 cpoy->coy, copy,
+craete->create
+crahses->crashes
+crasher->crash
+crashers->crashes
 crashs->crashes
+crated->created
```

Comment:
The word _crated_ is valid, this correction should be removed.

Reply:
I'll use `crated->created, crated,` JIC ?

Reply:
Sounds good.